examples/RingRodCase/ring_rod_post_processing.py

```diff
@@ -0,0 +1,436 @@
+import numpy as np
+import matplotlib
+
+matplotlib.use("Agg")  # Must be before importing matplotlib.pyplot or pylab!
+from matplotlib import pyplot as plt
+from matplotlib import cm
+from tqdm import tqdm
+from matplotlib.patches import Circle
+from typing import Dict, Sequence
+
+
+def plot_video(
+    rods_history: Sequence[Dict],
+    video_name="video.mp4",
+    fps=60,
+    step=1,
+    vis2D=True,
+    **kwargs,
+):
```

Comment:
if this is similar to other post-processing file functions I would recommend adding symlink or making this a part of utils in some way to avoid copy-pasting huge chunks of code.

Reply:
Yes, we have some similar post-processing files (not all). We can make a post-processing file in utils and make it a part of elastic. Also, it is possible to symlink similar ones. Which do you prefer @armantekinalp